frontend/src/features/admin-form/responses/ResponsesPage/storage/UnlockedResponses/ResponsesTable/ResponsesTable.tsx

```diff
@@ -99,8 +99,9 @@ const PAYMENT_COLUMNS: Column<ResponseColumnData>[] = [
       }
       return payments.payoutDate
     },
-    minWidth: 50,
+    minWidth: 150,
     width: 150,
+    disableResizing: true,
```

Comment:
Just for my own learning, but why can't the last element here be resized? 😅

Reply:
Technically they can, but the UX would be weird.

Extending the width will look like the table isn't growing as the additional width is cutoff by the scroll overflow.